english/game-on/game_on.md

```diff
@@ -0,0 +1 @@
+Suddenly he appears and passes the ball to me. I feel like an [nba](www.nba.com)star. I fell unstoppable.
```

Comment:
Even though that link works in web browsers (at least it does in mine), it will not work if you click it from the GitHub page. (You can try it yourself and see.) I believe you can fix that by adding `http://` to the beginning of the link.

Also, you may want to add an extra space between the link and "star" since right now the two words are smooshed together.